### ISSUE TITLE
TransitionNode fails if options.content is not provided

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/twixt#16]
Noted while working on https://github.com/phetsims/equality-explorer/issues/75.

TransitionNode:

```js
33      // {Node|null} - Optionally may have initial content
34      content: null,
```

But `content` is apparently not optional.  If you don't provide it, then your first attempted transition will fail in `startTransition` at line 225:

```js
      this.fromContent.pickable = false;
```

**Comment by pixelzoom** [comment]
Look OK via inspection.  But how did you test this?  Do we need some examples or unit tests?

**Comment by jonathanolson** [comment]
I manually tested creation, but unit tests would probably be good. I'll add those.

**Comment by pixelzoom** [comment]
In TransitionNode `startTransition`:

```js
      if ( this.fromContent ) {
        this.fromContent.pickable = false;
      }
      if ( this.toContent ) {
        this.toContent.pickable = false;
      }
```

So it's OK if they are _both_ null?  Why would someone do that?  Or should there be a check in `startTransition` that at least one of them is non-null?

**Comment by jonathanolson** [comment]
> So it's OK if they are both null? Why would someone do that? Or should there be a check in startTransition that at least one of them is non-null?

Being able to animate to `null` always sounds like a helpful property. Maybe it would be better to short-circuit a null-to-null transition and not actually animate at all?

**Comment by pixelzoom** [comment]
I agree that it's useful to animate to null if `fromContent` is defined.   But I don't understand what a null-to-null transition is, or why it's useful.

**Comment by pixelzoom** [comment]
Also related to https://github.com/phetsims/twixt/issues/11.
